src/jvm/test/org/jetbrains/kotlinx/lincheck_test/generator/ParamGeneratorsTests.kt

```diff
@@ -9,14 +9,16 @@
  * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-package org.jetbrains.kotlinx.lincheck.test.generator
+package org.jetbrains.kotlinx.lincheck_test.generator
 
-import junit.framework.Assert.*
+import junit.framework.Assert.assertTrue
 import org.jetbrains.kotlinx.lincheck.*
-import org.jetbrains.kotlinx.lincheck.annotations.*
-import org.jetbrains.kotlinx.lincheck.paramgen.*
-import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.*
-import org.junit.*
+import org.jetbrains.kotlinx.lincheck.annotations.Operation
```

Comment:
Please configure your IDE to ALWAYS use imports with "*" for Kotlin code. Update all the changed files correspondingly.